build.gradle

```diff
@@ -82,8 +82,11 @@ repositories {
 dependencies {
 
   compile group: 'cascading', name: 'cascading-hadoop2-mr1', version: cascadingVersion, changing: true
+  compile group: 'org.apache.commons', name: 'commons-csv', version: '1.2'
```

Comment:
As per my previous answer above. I suppose we can fetch String and unload the parsing to users